sdk/eventhub/azure-eventhub/azure/eventhub/_consumer_client.py

```diff
@@ -72,6 +72,8 @@ class EventHubConsumerClient(ClientBase):
      The failed internal partition consumer will be closed (`on_partition_close` will be called if provided) and
      new internal partition consumer will be created (`on_partition_initialize` will be called if provided) to resume
      receiving.
+    :keyword retry_mode: Fixed or exponential delay between attempts, default is exponential.
```

Comment:
@yunhaoling according to the[ issue/azure-core](https://github.com/Azure/azure-sdk-for-python/issues/8670), shouldn't we have the `retry_backoff_factor` and `retry_backoff_max` here as well? we don't have them defined in any of the docstring in sync/async producer/consumer clients.

Reply:
yep, those two are missing and we need to add them